acceptance/testdata/pr/pr-create-with-metadata.txtar

```diff
@@ -0,0 +1,26 @@
+# Use gh as a credential helper
+exec gh auth setup-git
+
+# Create a repository with a file so it has a default branch
+exec gh repo create $ORG/$SCRIPT_NAME-$RANDOM_STRING --add-readme --private
+
+# Defer repo cleanup
+defer gh repo delete --yes $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Clone the repo
+exec gh repo clone $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Prepare a branch to PR
+cd $SCRIPT_NAME-$RANDOM_STRING
+exec git checkout -b feature-branch
+exec git commit --allow-empty -m 'Empty Commit'
+exec git push -u origin feature-branch
+
+# Create the PR
+exec gh pr create --title 'Feature Title' --body 'Feature Body' --assignee '@me' --label 'bug'
+stdout2env PR_URL
+
+# Check the PR is indeed created
+exec gh pr view $PR_URL
+stdout 'assignees:\t.*$'
```

Comment:
I can appreciate this might be trickier to verify without having the script `exec gh api /user --jq .login` as the user's local username might not match. 🤔

Reply:
Actually this should probably be:

```
stdout 'assignees:\t.+$`
```

To ensure there is at least a character there. Thanks for bringing my attention to it!

We could check the actual content with the stdout of `gh api` but tbh mostly what I want to check is that the platform accepted an assignment at all.